src/styles/variables.scss

```diff
@@ -13,13 +13,34 @@ $color-red: #f95252;
 $color-white: #ffffff;
 $color-yellow: #e5c650;
 
-$size-xsmall: 11px;
+$size-xxsmall: 8px;
+$size-xsmall: 12px;
 $size-small: 16px;
-$size-base: 23px;
-$size-large: 32px;
-$size-xlarge: 45px;
-$size-xxlarge: 64px;
+$size-base: 24px;
+$size-large: 40px;
+$size-xlarge: 64px;
+$size-xxlarge: 84px;
 
 $max-tablet-size: 768px;
 
-$grid-size: 992px;
\ No newline at end of file
+$grid-size: 992px;
+
+$Logo-font-size-small: 12px;
+$Logo-padding: 2px;
+
+// Typography
+
+$fontSize-xsmall: 12px;
+$fontSize-small: 16px;
+$fontSize-medium: 20px;
+$fontSize-large: 28px;
+$fontSize-xlarge: 36px;
+$lineHeight-xsmall: 16px;
+$lineHeight-small: 20px;
+$lineHeight-medium: 28px;
+$lineHeight-large: 36px;
+$lineHeight-xlarge: 52px;
+$fontFamily: 'proxima-nova', helvetica arial, sans-serif;
+$fontWeight-light: 300;
+$fontWeight-regular: 400;
+$fontWeight-bold: 700;
```

Comment:
The variable naming is very, very inconsistent. If the components are `Uppercase`, why are these `camelCase` and why are some even all `downcase`?  [Example](https://github.com/subvisual/2017.mirrorconf.com/blob/master/css/Theme/Global.scss)

Reply:
Will fix this later :+1:

Reply:
Fixed.